mlflow/utils/conda.py

```diff
@@ -186,7 +186,7 @@ def get_or_create_conda_env(conda_env_path, env_id=None, capture_output=False):
                         capture_output=False,
                     )
             except Exception:
-                # swallow exception raised during cleaning env.
+                _logger.warning(f"Removing conda env '{project_env_name}' failed.", exc_info=True)
```

Comment:
Can we include the exception in the warning message?

Reply:
I missed line 186. `capture_output` is set to False. I think we don't need `exc_info=True`.

```suggestion
                _logger.warning(f"Removing conda env '{project_env_name}' failed.")
```